### DISCUSSION TITLE
sql too long

### DISCUSSION THREAD

**Discussion by MakenChan** [apache/skywalking#7508] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/20518466/130188955-f8a5e1b7-83ed-4a86-9c7d-8e6a7dcf7c56.png)
![image](https://user-images.githubusercontent.com/20518466/130189027-c1f740a4-f9dd-4158-8fb9-7ba5c7cc4940.png)

**Comment by wu-sheng** [comment, marked as answer]
Could you make yourself clear? What is the question?

**Reply by MakenChan** [reply]
The SQL statement is too long, resulting in incomplete display

**Reply by wu-sheng** [reply]
Yes, we were facing ~10k length SQL, if we keep it all, it easily makes your agent OOM. So we set up the limitation. Read the agent configuration list, the threshold is there.
